### ISSUE TITLE
Replacement data/readyplayerme.onim

### ISSUE THREAD

**Issue by aiaustin** [nessaki/Onigiri#2]
One little change I suggest if you use Onigiri. This change was made in Bento Buddy in September 2022 at my suggestion after nessaki made this fork.

The data/readyplayerme.onim mapping preset is not enough.But Ready Player Me avatars have a Mixamo compatible skeleton. You can just copy mixamo_no_prefix.onim to readyplayerme.onim and that mapping will work better.

**Comment by aiaustin** [comment]
I have created a new fork with the improvement to readyplayerme.onim (copy of mixamo_no_prefix.onim) and addition of ReadyPlayerMe-T-Pose-Splayed-Hands-Only.bpl. I also updated Onigiri.zip for these changes.  maybe nessaki could consider merging these changes into the main branch and creating a new release package (3.5c?).

Would also need a small change to README.md to update version number mentioned there.
